packages/components/addon/components/hds/pagination/nav/ellipses.hbs

```diff
@@ -0,0 +1,3 @@
+<li class="hds-pagination-nav__page-item" ...attributes>
+  <div class="hds-pagination-nav__ellipses">...</div>
```

Comment:
I used 3 dots vs. an ellipses character because visually it more closely matched the Figma mocks. (I'm fine with using either one.)

Reply:
Super uber nitpick: should be `ellipsis` and not `ellipses`, which is plural? 
https://www.merriam-webster.com/dictionary/ellipsis

Reply:
I'll change it, thanks